sample.txt

```diff
@@ -1,3 +1,5 @@
 This file is added by Venkat Putrevu.
 
 Student#1: Venkat Putrevu.
+Stufent#2: Narsi Kota.
+Module 2 Git Hub exercise.
```

Comment:
Please correct spelling (I know this is intentional to test PR process).   Stufent#2:   to Student#2:.